heliopy/data/mms.py

```diff
@@ -49,7 +50,8 @@ def _validate_data_rate(data_rate):
                 data_rate, allowed_rates))
 
 
-def available_files(probe, instrument, starttime, endtime, data_rate=''):
+def available_files(probe, instrument, data_rate, starttime, endtime,
+                    product_string=''):
```

Comment:
It looks like the order has changed here, `data_rate` should go back to where it was before (after `endtime`)